File: lib/src/main/java/com/tuti/api/data/Fees.java
package com.tuti.api.data;


import com.google.gson.annotations.SerializedName;
import java.io.Serializable;

public class Fees implements Serializable {

    @SerializedName("mohe_fees")
    float moheFees;
    @SerializedName("p2p_fees")
    float p2pFees;
    @SerializedName("custom_fees")
    float customFees;
    @SerializedName("special_payment_fees")
    float specialPaymentFees;

    public float getMoheFees() {
        return moheFees;
    }

    public float getP2pFees() {
        return p2pFees;
    }

    public float getCustomFees() {
        return customFees;
    }

    public float getSpecialPaymentFees() {
        return specialPaymentFees;
    }

}

File: lib/src/main/java/com/tuti/api/data/Fees.kt
package com.tuti.api.data

import kotlinx.serialization.SerialName

@kotlinx.serialization.Serializable
data class Fees(
        @SerialName("mohe_fees")
        val moheFees: Float = 0f,

        @SerialName("p2p_fees")
        val p2pFees: Float = 0f,

        @SerialName("custom_fees")
        val customFees: Float = 0f,

        @SerialName("special_payment_fees")
        val specialPaymentFees: Float = 0f,
)